src/main/java/org/zalando/problem/spring/web/advice/StatusConverter.java

```diff
@@ -68,7 +67,22 @@
     default StatusType convert(final HttpStatus httpStatus) {
         final int statusCode = httpStatus.value();
         @Nullable final StatusType statusType = INDEX.get(statusCode);
-        return statusType == null ? new UnknownStatus(statusCode) : statusType;
+        return statusType != null ? statusType : new StatusType() {
```

Comment:
I don't mind the anonymous implementation. Alternative if it would be its own class, maybe `HttpStatusTypeAdapter` or something?

I did take a look again and realized that in the problem library we only have the integer status code at hand, hence the need for UnknownStatus (including unknown family and reason phrase). In this case, since we have a concrete status, maybe we don't even need the lookup? We could just always return that adapter. What do you think? Would simplify the implementation a bit, I'd assume.